### PR TITLE
Adding textAreaPage as scaffolding component

### PR DESCRIPTION
**Pull request by grantrobson**

### CODE REVIEW

File: .g8/textAreaPage/generated-test/controllers/$package$/$className$ControllerSpec.scala [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+      running(application) {
+        val request =
+          FakeRequest(POST, postRoute).withFormUrlEncodedBody(("value", ""))
+
+        val result = route(application, request).value
+        val updatedAnswers = emptyUserAnswers.set($className$Page, validValue).success.value
+
+        status(result) mustEqual SEE_OTHER
+        redirectLocation(result).value mustEqual $className$Page.navigate(waypoints, emptyUserAnswers, updatedAnswers).url
```
Comment (cyeniceri) [inline .g8/textAreaPage/generated-test/controllers/$package$/$className$ControllerSpec.scala:137]: It is better to verify mock at the end 
`        verify(mockUserAnswersCacheConnector, times(1)).save(any(), any(), any())(any(), any())
`

File: .g8/textAreaPage/generated-test/controllers/$package$/$className$ControllerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+      }
+    }
+
+    "must return bad request when invalid data is submitted" in {
+      when(mockUserAnswersCacheConnector.save(any(), any(), any())(any(), any()))
```
Comment (cyeniceri) [inline .g8/textAreaPage/generated-test/controllers/$package$/$className$ControllerSpec.scala:142]: this mock is not required here

File: .g8/textAreaPage/generated-test/controllers/$package$/$className$ControllerSpec.scala [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+        applicationBuilder(userAnswers = Some(emptyUserAnswers), extraModules)
+          .build()
+
+      running(application) {
+        val request =
+          FakeRequest(POST, postRoute).withFormUrlEncodedBody(("value", "A" * 151))
+
+        val result = route(application, request).value
+
+        status(result) mustEqual BAD_REQUEST
```
Comment (cyeniceri) [inline .g8/textAreaPage/generated-test/controllers/$package$/$className$ControllerSpec.scala:155]: we can verify that save method will never being called
`verify(mockUserAnswersCacheConnector, never()).save(any(), any(), any())(any(), any())`

File: .g8/textAreaPage/generated-test/controllers/$package$/$className$ControllerSpec.scala (hunk 1 above)
Comment (grantrobson) [reply .g8/textAreaPage/generated-test/controllers/$package$/$className$ControllerSpec.scala:138]: Good point. Sorted and pushed.

File: .g8/textAreaPage/generated-test/controllers/$package$/$className$ControllerSpec.scala (hunk 3 above)
Comment (grantrobson) [reply .g8/textAreaPage/generated-test/controllers/$package$/$className$ControllerSpec.scala:154]: Good point. Sorted and pushed.

**Review by cyeniceri — APPROVED** [review]

**Pull request merged**